src/NzbDrone.Core/Notifications/Webhook/WebhookEpisodeFile.cs

```diff
@@ -0,0 +1,31 @@
+﻿using NzbDrone.Core.MediaFiles;
+using NzbDrone.Core.Tv;
+using System.Collections.Generic;
+using System.IO;
+
+namespace NzbDrone.Core.Notifications.Webhook
+{
+    internal class WebhookEpisodeFile
+    {
+        public WebhookEpisodeFile(Series series, EpisodeFile episodeFile)
+        {
+            Id = episodeFile.Id;
+            RelativePath = episodeFile.RelativePath.ToString();
+            FullPath = Path.Combine(series.Path, episodeFile.RelativePath).ToString();
+            SeasonNumber = episodeFile.SeasonNumber;
+            Quality = episodeFile.Quality.Quality.Name;
+            QualityVersion = episodeFile.Quality.Revision.Version;
+            ReleaseGroup = episodeFile.ReleaseGroup ?? string.Empty;
+            SceneName = episodeFile.SceneName ?? string.Empty;
```

Comment:
I'm a bit torn on this, @markus101 what do you think? should we send `''`back when these are empty or keep them `null`?